src/components/App/App.js

```diff
@@ -1,22 +1,50 @@
 import './App.css';
 import Header from '../Header/Header';
 import MovieCardContainer from '../MovieCardContainer/MovieCardContainer';
+import Error from '../Error/Error';
 import SingleMovie from '../SingleMovie/SingleMovie';
 import { useState, useEffect } from 'react';
 
 function App() {
   const [movieData, setMovieData] = useState([]);
   const [showMovie, setShowMovie] = useState(false);
-  const [clickedMovie, setClickedMovie] = useState([])
+  const [clickedMovie, setClickedMovie] = useState([]);
+  const [error, setError] = useState(false);
 
   useEffect(() => {
     fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
-    .then(response => response.json())
-    .then(data => setMovieData(data.movies))
+    .then(response => {
+      if (!response.ok) {
+        let errMessage = handleErrors(response);
+        setError(errMessage)
+      }
+      return response.json()
+    })
+    .then(data => {
+      console.log('data not catching')
+      setMovieData(data.movies)
+      return data
+    })
+    .catch(err => {
+      console.log('error: ', err)
```

Comment:
Consider what you want to do in your catch block.  If there is an error, what do you want to display to the user?  How might you approach doing that?